### PR TITLE
Model updates

### PR DESCRIPTION
**Pull request by pgier**
NCL-442, NCL-445, NCL-464

### CODE REVIEW

File: common/pom.xml [hunk 1]
```diff
@@ -14,51 +14,44 @@
...
       <groupId>org.jboss</groupId>
       <artifactId>jboss-common-core</artifactId>
-       <exclusions>
-           <exclusion>
-               <groupId>org.jboss.logging</groupId>
-               <artifactId>jboss-logging-spi</artifactId>
-           </exclusion>
-       </exclusions>
-    </dependency>
-        <dependency>
```
Comment (slaskawi) [inline common/pom.xml]: formatting

File: pnc-rest/src/main/java/org/jboss/pnc/rest/restmodel/ProductVersionRest.java [hunk 2]
```diff
@@ -48,6 +57,30 @@ public void setVersion(String version) {
...
+    public boolean isReleased() {
+        return released;
+    }
+
+    public void setReleased(boolean released) {
+        this.released = released;
+    }
+
+    public boolean isSupported() {
+        return supported;
```
Comment (slaskawi) [inline pnc-rest/src/main/java/org/jboss/pnc/rest/restmodel/ProductVersionRest.java]: #toProductVersion methods also need to be modified

Comment (pgier) [inline pnc-rest/src/main/java/org/jboss/pnc/rest/restmodel/ProductVersionRest.java]: done

File: common/pom.xml (hunk 1 above)
Comment (pgier) [inline common/pom.xml]: What is the formatting issue, it looks ok to me?

Comment (slaskawi) [inline common/pom.xml]: aaaah I see Github adds this `-`. Sorry my bad...

**Pull request merged**